game/src/main/org/apollo/game/message/impl/PrivateChatMessage.java

```diff
@@ -60,7 +31,7 @@ public String getChat() {
 	 *
 	 * @return The username.
 	 */
-	public String getUsername() {
+	public final String getUsername() {
```

Comment:
Oops, I meant only the accessors in the abstract chat message class to indicate that they should not be overridden. Could you undo all of these except for the ones in the abstract class?